src/client/js/chart-options.jsx

```diff
@@ -73,6 +75,10 @@ class ChartOptions extends React.Component {
                   {topicMenuItems}
                 </Select>
               </FormControl>
+              <Tooltip title="Select the meter you would like to receive data from. You may
+                select multiple to overlap data.">
+	            <InfoIcon />
+	          </Tooltip>
```

Comment:
Can you fix the indentation of this? Every new level should be indented by 2 extra spaces. Also, put the Tooltip title text all on one line. It should look like this:

<Tooltip title="Select the meter you would like to receive data from. You may select multiple to overlap data.">
  <InfoIcon />
</Tooltip>